src/Javis.jl

```diff
@@ -392,6 +394,7 @@ function javis(
     nothing
 end
 
+export draw_grid, zero_lines
```

Comment:
Can/Should be done in `backgrounds.jl` I think.

Reply:
Oh! Didn't know that by including it in the main package, the export would still be executed. Done!